content/en/docs/ops/deployment/deployment-models/index.md

```diff
@@ -266,33 +261,10 @@ cluster.
 A cluster like this one, with its own local control plane, is referred to as
 a {{< gloss >}}primary cluster{{< /gloss >}}.
 
-Multicluster deployments can have any number of primary clusters. Each primary
-cluster receives its configuration (i.e. `Service`/`ServiceEntry`,
-`DestinationRule`, etc.) from the Kubernetes API Server residing in the same
-cluster. Each primary cluster, therefore, has an independent source of
-configuration.
-
-This duplication of configuration across primary clusters does require
-additional steps when rolling out changes, however. Large production
-systems may automate this process with tooling, such as CI/CD systems, in
-order to manage configuration rollout.
-
-There may be use cases where having different configurations among primary
-clusters is desirable. For example, this affords the ability to canary
-configuration changes in a sub-section of the mesh controlled by a
-given primary cluster. Alternatively, visibility of a service could be
-restricted to part of the mesh, helping to establish service-level isolation.
-
-A {{< gloss >}}remote cluster{{< /gloss >}} refers to a cluster that contains
-user workloads but no control plane. Remote clusters rely on a control plane
-residing outside of the cluster (e.g. in a primary cluster). In this case, the
-control plane must be made accessible to the remote cluster.
-
-This can be achieved by exposing the control plane through an Istio Gateway.
-Cloud vendors may provide options, such as internal load balancers, for
-providing this sort of capability without exposing the control plane on the
-public internet. Such an option, if it exists and meets your requirements,
-will likely be the best choice.
+Multicluster deployments can also share control plane instances. In this case,
```

Comment:
This paragraph now seems like it's talking about two separate things at once (primary and remote). The old wording intentionally tried to describe in entirety the concept of primary clusters before mentioning remotes. I think that's a better approach in general, since it stays on topic. Perhaps subsections would help, where we first talk about primaries and then remotes?

Reply:
That was intentional. The section is talking control planes in multicluster configurations, which are comprised of two kinds of clusters. First define the terms with the pictures, then talk about how they work. 

I had considered splitting the rest (after the overview) into separate sections, but concluded there wasn't really enough material to justify it.

Reply:
I actually think it might be worth it. The remote section can talk about how to connect to a primary. Maybe give it a try and see if it works?

Reply:
Looking at it, this section won't flow well with subsections. Let's leave it as is for now. We can reconsider next time we reorg this doc 😄